assignment4/sass/pages/_details.scss

```diff
@@ -0,0 +1,348 @@
+			    border-right: none;
+			    border-top: none;
+			    display: block;
+			    margin-bottom: 20px;
+			    width: 91%;
+			    border-bottom: 2px solid;
+			    border-color: blue;
+    		}
+
+			button {
+				float: right;
+			    margin-right: 44px;
+			    background-color: #7E90e0;
+			    padding: 8px;
+			    border-color: #7e90e0;
+			    margin-top: -10px;
+			}
+			a {
+				text-decoration: none;
+			}
+		}
+		.user-name {
+		    margin-top: 15px;
+			width: 100%;
+			font-size: 14px;
+			span {
+				padding: 4px;
+			    background-color: #868789;
+			    border-radius: 23px;
+			    margin-right: 15px;
+			    margin-bottom: 10px;
+			    padding-right:0px;
+			    span {
+		    		background-color: red;
+				    padding: 1px;
+				    padding-left: 6px;
+				    margin-left: 5px;
+			    }
+
+			}
+			a {
+				text-decoration: none;
+			    color: white;
+			    font-size:10px;
+
+			}
+		}
+		
+
+	}
+
+	.test-report-2 {
+		@extend .test-report;
+		.user-detail {
+			label {
+				color:grey;
+			}
+			input[type="text"] {
+				border-color:grey;
+			}
+		}
+	}
+
+	.tag-summary {
+		@extend .test-report-2;
+	}
+
+}
+
+@media only screen and (max-width:1294px) and (min-width:1234px) {
+	.middle-container {
+		width:81.1%;
+	}
+	.detail {
+		padding-top:12px;
+	}
+	.user-detail {
+		margin-top:-123px;
+	}
+
+}
+
+@media only screen and (max-width:1233px) and (min-width:1167px) {
+	.middle-container {
+		width:80%;
+	}
+	.detail {
+		padding-top:21px;
+	}
+	.user-detail {
+		margin-top:-123px;
+	}
+
+}
+
+@media only screen and (max-width:1166px) and (min-width:1113px) {
+	.middle-container {
+		width:79%;
+	}
+	.detail {
+		padding-top:26px;
+	}
+	.user-detail {
+		margin-top:-123px;
+	}
+
+}
+
+@media only screen and (max-width:1112px) and (min-width:1063px) {
+	.middle-container {
+		width:78%;
+	}
+	
+	.user-detail {
+		margin-top:-120px;
+	}
+	.detail {
+		padding-top:35px;
+	}
+
+	.subscription {
+		font-size:15px;
+	}
+
+}
+
+@media only screen and (max-width:1062px) and (min-width:1024px) {
+	.middle-container {
+		width:77%;
+	}
+	
+	.user-detail {
+		margin-top:-110px;
+	}
+	.detail {
+		padding-top:40px;
+	}
+
+	.subscription {
+		font-size:15px;
+	}
+	.user-name {
+		font-size:25px;
+	}
+	.org-name {
+		font-size: 20px;
+	}
+
+}
+
+@media only screen and (max-width:1023px) and (min-width:768px)  {
+	.header {
+		.logo {
+			width:17%;
+			margin-top:10px;
+			margin-left:35%;
+		}
+
+		.fa {
+		    margin-left: -45%;
+    		margin-top: 2.2%;
+		}
+
+		.searchbar {
+			margin-left:6%;
+		}
+	}
+	.sidebar {
+		display:none;
+	}
+	.middle-container {
+		float:none;
+		width:100%;
+	}
+	.detail {
+		padding-top:58px;
+
+		.report-img {
+			img {
+				margin-bottom:15px;
+			}
+		}
+	}
+	.subscription {
+		font-size:15px;
+	}
+
+}
+
+@media only screen and (max-width:1023px) and (min-width:900px){
+	.headre {
+		.logo {
+			width:15%;
+		}
+	}
+}
+
+@media only screen and (max-width:767px) and (min-width:500px)  {
```

Comment:
@priyanka1110  Media query is not working for mobile. 500px I have added for testing.  For some other screen size also some media query was not working.  Today I got to know that I need to add `<meta name='viewport' content='width=device-width, initial-scale=1, maximum-scale=1.0' />` .